assets/constants.ts

```diff
@@ -5187,6 +5187,8 @@ export = {
 
   "NUM_QUESTIONS_PER_PAGE": 10,
 
+  "TEMP_SKILL_ENTITY": "tmp_skills",
```

Comment:
No, don't do this. It's really not an entity at all, so you shouldn't call it one. 

Can you figure out the correct concept for this?

Reply:
Done, removed this. Now, saving the images in the 'skill' folder itself but under a temporary ID and then moving into the correct ID's folder, once it is created.